exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -338,16 +346,29 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
             extensions: res.extensions,
           };
 
+          let debugMessage = 'A cache hit occured';
           if (
             policy === 'cache-and-network' ||
             (policy === 'cache-first' && outcome === 'partial')
           ) {
+            debugMessage +=
+              ' but is being retried due to the policy or outcome';
```

Comment:
```suggestion
              outcome === 'partial' && policy === 'cache-first' ? ' but is being retried due to the result being partial.' : ' but is being retried due to "cache-and-network.';
```

I hope this get DCE'd since pure strings are quite horrible to have in bundle